audio/whisper/Portfile

```
@@ -24,36 +24,42 @@ output.wav` to convert the file first.
 "
 
 checksums           whisper.cpp-${version}.tar.gz \
-                    rmd160  26b0b29e46c990b13c9d10d95c11aa429fc8d56d \
-                    sha256  da7988072022acc3cfa61b370b3c51baad017f1900c3dc4e68cb276499f66894 \
-                    size    5088298
+                    rmd160  211e55a336132a878471382d8002425f069db683 \
+                    sha256 2fda42b57b7b8427d724551bd041616d85401fb9382e42b0349132a28920a34f \
```

Can we nudge the `sha256` sum over to align with everything else?

```suggestion
sha256  2fda42b57b7b8427d724551bd041616d85401fb9382e42b0349132a28920a34f \
```